File: Iniciante/1036/1036.py
import math

float_str = input()
a, b, c = float_str.split()

a = float(a)
b = float(b)
c = float(c)

delta = b * b - (4 * a * c)

if delta > 0 and a != 0:
    x1 = (-b + math.sqrt(delta)) / (2 * a)
    x2 = (-b - math.sqrt(delta)) / (2 * a)
    print("R1 = %.5f" %x1)
    print("R2 = %.5f" %x2)
else:
    print("Impossivel calcular")

File: Iniciante/Pagina 01/1036/1036.c
#include <stdio.h>
#include <math.h>

int main() {
    double a, b, c, x1, x2, delta;

    scanf("%lf", &a);
    scanf("%lf", &b);
    scanf("%lf", &c);

    delta = b * b - (4 * a * c);
    x1 = (-b + sqrt(delta)) / (2 * a);
    x2 = (-b - sqrt(delta)) / (2 * a);
    
    if (delta > 0 && a != 0) {
        printf("R1 = %.5lf\n", x1);
        printf("R2 = %.5lf\n", x2);
    } else {
        printf("Impossivel calcular\n");
    }

    return 0;
}